game/src/game_thread.rs

```diff
@@ -106,6 +106,7 @@ pub enum PlayerLoseType {
 pub struct GameThreadResults {
     // Index by ingame player id
     pub victory_state: [u8; 8],
+    pub race: [Option<u8>; 8],
```

Comment:
This can be just u8; and the `player.player_id` indirection below isn't needed as that field itself is redundant - `players[i].player_id == i` always.

(Or well, the field is only useful if code would take a pointer to a single struct in the array)

Reply:
Okay that's what I thought but I couldn't recall 100%. Adjusted to the simpler approach